util/topgen/templates/chiplevel.sv.tpl

```diff
@@ -407,77 +390,63 @@ module chip_${top["name"]}_${target["name"]} (
   // TODO: generalize this USB mux code and align with other tops.
 
   // Only use the UPHY on CW310, which does not support pin flipping.
+  logic usb_dp_pullup_en;
+  logic usb_dn_pullup_en;
+  logic usb_rx_d;
+  logic usb_tx_d;
+  logic usb_tx_se0;
+  logic usb_tx_mode_se;
+  logic usb_suspend;
+  logic usb_rx_enable;
```

Comment:
These are all the new inter-sig signals. They have been converted from DIOs through the pinmux because they would only go to pads when a transceiver / differential receiver is off-die (such as this FPGA case).